src/fsharp/ParseAndCheckInputs.fs

```diff
@@ -686,11 +686,21 @@ let TypeCheckOneInputEventually (checkForErrors, tcConfig: TcConfig, tcImports:
               let conditionalDefines =
                   if tcConfig.noConditionalErasure then None else Some (tcConfig.conditionalCompilationDefines)
 
+              let hadSig = rootSigOpt.IsSome
```

Comment:
I think I know a way where we don't have to modify TypeCheckOneInputEventually.

Reply:
There wasn't a way to remove `quickCheck`. Instead, I renamed it to `skipImplIfSigExists`.